src/about.js

```diff
@@ -10,17 +12,34 @@ const workstation = require('electron').remote.require('./chef_workstation.js');
 // we eventually want most of these to be links to documentation on the Chef website. The license should
 // always display from local source but we also want these 3 links to all behave similarly.
 function openLicense() {
-  licensePath = path.join('file://', __dirname, '../assets/html/license.html');
+  licensePath = path.join('file://', helpers.getAssetsDir(), 'html/license.html');
   shell.openExternal(licensePath)
 }
 
+// If we do not have external internet connectivity to the hosted release notes point people
+// at the ones included into the package.
 function openReleaseNotes() {
-  notesPath = path.join('file://', __dirname, '../assets/html/release_notes.html');
-  shell.openExternal(notesPath)
+  let cwVersion = workstation.getVersion();
+  let remoteReleaseNotes = `https://packages.chef.io/release-notes/stable/chef-workstation/${cwVersion}.md`
+  if (cwVersion == "development") {
+    shell.openExternal(remoteReleaseNotes);
+  } else {
+    // This displays raw markdown now but we will update it to display rendered markdown
+    // once we have a location for that
+    https.get(remoteReleaseNotes, function(res) {
+      shell.openExternal(remoteReleaseNotes);
```

Comment:
Does this mean we're still defaulting to opening about-links in the user's default web browser? And then our fallback if we can't retrieve the URL is to display local file content in an Electron window?

Reply:
Nope, both are actually open in the default Web Browser. Electro is a bit slick here and detect the extension of the file and opens the right applications, if this were a plain text file, it will probably use **TextEdit** as the default. (for Mac users)

Reply:
So the `BrowserWindow` below in the error condition is also the external default browser?

Reply:
yup. you can read about this API https://electronjs.org/docs/api/browser-window

Reply:
Wait, no, you are right. This will be a Electron Window. Apologies.

Reply:
Alrightie! Do we need to update the comment block for the `openLicense` function? It still states that we want the same behavior for all three links and now it looks like we're OK with letting that behavior differ.

Reply:
Updated them! ⭐️

Reply:
Updated it ⭐️